commandline.c

```diff
@@ -57,14 +58,20 @@ int main(int argc, char* argv[])
 		goto out;
 	}
 
-	if (strlen(argv[1]) < 3)
-		iface = socket_find_iface_by_index(argv[1]);
+	if (argc == 5) {
```

Comment:
no, use getopt for things like this.

Reply:
I've already used getopt_long with

    #include <getopt.h>

using mingw and it worked rather fine. And yes, we even require that it works under Windows. This is also the reason why we (at least) do an automated Windows compile test. Also your PR it automatically triggered it